packages/enhanced/src/lib/container/runtime/FederationRuntimePlugin.ts

```diff
@@ -151,7 +151,10 @@ class FederationRuntimePlugin {
             // TODO: maybe set this variable as constant is better https://github.com/webpack/webpack/blob/main/lib/config/defaults.js#L176
             entryItem.import = ['./src'];
           }
-          if (!entryItem.import.includes(entryFilePath)) {
```

Comment:
We should not modify /enhanced in order to fix next.js 

If you need to do this, we should add a hook or some option that we can pass down from next.js instead.

/enhanced needs to remain as pure as it can and not patch framework specific issues but rather bugs in webpack etc (like single runtime chunk) X

Reply:
Yes, maybe we can add "exclude" option to allow users control  whether add our federation runtime entry file.

Reply:
@ScriptedAlchemy @ahabhgk  How about adding `exclude` options to allow users control whether add our federation runtime entry file in rspack/webpack ?

Reply:
yeah im good with that